### ISSUE TITLE
Failed to download https://nightlies.plex.tv/directdl/plex-dependencies/plexmediaplayer-qt/140/konvergo-qt--release-14044be.tbz2.sha.txt

### ISSUE THREAD

**Issue by Enigma0** [plexinc/plex-media-player#592]
```
-- Performing Test HAVE_FLAG_mmacosx_version_min=10.9
-- Performing Test HAVE_FLAG_mmacosx_version_min=10.9 - Failed
-- Performing Test HAVE_FLAG_fuse_ld=gold
-- Performing Test HAVE_FLAG_fuse_ld=gold - Success
-- Not using bundled deps
-- Dependency hash is: 14044be
-- Downloading https://nightlies.plex.tv/directdl/plex-dependencies/plexmediaplayer-qt/140/konvergo-qt--release-14044be.tbz2.sha.txt to konvergo-qt--release-14044be.tbz2.sha.txt...
CMake Error at CMakeModules/FetchDependencies.cmake:59 (message):
  Failed to download
  https://nightlies.plex.tv/directdl/plex-dependencies/plexmediaplayer-qt/140/konvergo-qt--release-14044be.tbz2.sha.txt:
  "HTTP response code said error"

    Trying 104.20.6.9...

  TCP_NODELAY set

  Connected to nightlies.plex.tv (104.20.6.9) port 443 (#0)

  ALPN, offering http/1.1

  Cipher selection:
  ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH

  successfully set certificate verify locations:

    CAfile: /etc/ssl/certs/ca-certificates.crt
    CApath: /etc/ssl/certs

  TLSv1.2 (OUT), TLS header, Certificate Status (22):

  [5 bytes data]

  TLSv1.2 (OUT), TLS handshake, Client hello (1):

  [512 bytes data]

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Server hello (2):

  [106 bytes data]

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Certificate (11):

  [2489 bytes data]

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Server key exchange (12):

  [333 bytes data]

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Server finished (14):

  [4 bytes data]

  [5 bytes data]

  TLSv1.2 (OUT), TLS handshake, Client key exchange (16):

  [70 bytes data]

  [5 bytes data]

  TLSv1.2 (OUT), TLS change cipher, Client hello (1):

  [1 bytes data]

  [5 bytes data]

  TLSv1.2 (OUT), TLS handshake, Finished (20):

  [16 bytes data]

  [5 bytes data]

  TLSv1.2 (IN), TLS change cipher, Client hello (1):

  [1 bytes data]

  [5 bytes data]

  TLSv1.2 (IN), TLS handshake, Finished (20):

  [16 bytes data]

  SSL connection using TLSv1.2 / ECDHE-RSA-AES128-GCM-SHA256

  ALPN, server accepted to use http/1.1

  Server certificate:

   subject: C=US; ST=California; L=Los Gatos; O=Plex, Inc; CN=*.plex.tv
   start date: Feb 10 00:00:00 2017 GMT
   expire date: May 10 12:00:00 2020 GMT
   subjectAltName: host "nightlies.plex.tv" matched cert's "*.plex.tv"
   issuer: C=US; O=DigiCert Inc; CN=DigiCert SHA2 Secure Server CA
   SSL certificate verify ok.

  [5 bytes data]

  GET
  /directdl/plex-dependencies/plexmediaplayer-qt/140/konvergo-qt--release-14044be.tbz2.sha.txt
  HTTP/1.1

  Host: nightlies.plex.tv

  User-Agent: curl/7.51.0

  Accept: */*

  

  [5 bytes data]

  The requested URL returned error: 404 Not Found

  Curl_http_done: called premature == 1

  Closing connection 0

  [5 bytes data]

  TLSv1.2 (OUT), TLS alert, Client hello (1):

  [2 bytes data]

Call Stack (most recent call first):
  CMakeModules/FetchDependencies.cmake:117 (get_content_of_url)
  CMakeModules/QtConfiguration.cmake:9 (download_deps)
  CMakeLists.txt:60 (include)


-- Configuring incomplete, errors occurred!
See also "/home/[user]/git/pmp/plex-media-player/build/CMakeFiles/CMakeOutput.log".
See also "/home/[user]/git/pmp/plex-media-player/build/CMakeFiles/CMakeError.log".
```

**Comment by LongChair** [comment]
We use a TOKEN for the deps that are dynamically downloaded , and i don't think that this token is meant to be public.

Alternatively you can install qt and specify the root path to cmake with -DQTROOT=<root qt path>

**Comment by Enigma0** [comment]
Was trying to leave it out for this issue: https://github.com/plexinc/plex-media-player/issues/590

It does build when I put `-DQTROOT` back in.

**Comment by wm4** [comment]
Actually, our Qt deps are public. This is a different error due to cmake somehow producing a broken URL.

**Comment by LongChair** [comment]
this should have been fixed.

**Comment by azariah001** [comment]
This is still an issue.

````
  GET
  /directdl/plex-dependencies/plexmediaplayer-qt/206/konvergo-qt--release-206d4d1.tbz2.sha.txt
  HTTP/1.1

  Host: nightlies.plex.tv

  User-Agent: curl/7.58.0

  Accept: */*
````

Trying to compile on Linux Mint 19.

I've tried accessing the nightlies.plex.tv directly and it's prompting for auth at root, not authorised at `/directdl/`, directory not found for every north of `/directdl/plex-dependencies/`....

Why can't we just have a standard Linux package like we used to? Or an AppImage would be a great middle ground in that regard. This is ridiculously complicated, especially as paying user.
